pkg/reconciler/parallel/parallel.go

```diff
@@ -22,7 +22,8 @@ import (
 
 	"go.uber.org/zap"
 	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/equality"
+
+	//"k8s.io/apimachinery/pkg/api/equality"
```

Comment:
If we don't need the import, we can just remove it
```suggestion
```

Reply:
okay, I'll also try to deal with the test coverage , thanks @pierDipi